mm2src/coins/utxo/utxo_common.rs

```diff
@@ -3252,9 +3251,6 @@ pub fn min_tx_amount(coin: &UtxoCoinFields) -> BigDecimal {
 }
 
 pub fn min_trading_vol(coin: &UtxoCoinFields) -> MmNumber {
-    if coin.conf.ticker == "BTC" {
-        return MmNumber::from(MIN_BTC_TRADING_VOL);
-    }
     let dust_multiplier = MmNumber::from(10);
     dust_multiplier * min_tx_amount(coin).into()
```

Comment:
I would suggest to use the `min_tx_amount` without `dust_multiplier`, that is about 1000 sats. But need @cipig opinion on this and how it would work for all utxos.

Reply:
There is also this issue regarding 10x dust min_volume: https://github.com/KomodoPlatform/komodo-defi-framework/issues/1567. The idea there was to ensure swap min_volume > txfee of the HTLC spend tx

Reply:
ok, will leave it as is for now as this PR is to solve BTC problem only. https://github.com/KomodoPlatform/komodo-defi-framework/issues/1567 should be fixed in a separate PR.